src/model/profile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+import mongoose from 'mongoose';
```

Comment:
No points lost since the lab didn't explicitly advise against it, but try to avoid using the same schema names as we used in class.